pyccl/tests/test_power_mu_sigma_sigma8norm.py

```diff
@@ -29,6 +30,22 @@ def test_power_mu_sigma_sigma8norm(tf):
             ccl.linear_matter_power(cosmo_musig, 1e-4, a))
         assert np.allclose(pk_rat, gfac)
 
+    if tf == 'boltzmann_isitgr':
+        # make sure P(k) ratio is right
+        a = 0.8
+        gfac = (
+            ccl.growth_factor(cosmo, a) / ccl.growth_factor(cosmo_musig, a))**2
+        pk_rat = (
+            ccl.linear_matter_power(cosmo, 1e-4, a) /
+            ccl.linear_matter_power(cosmo_musig, 1e-4, a))
+        assert np.not_equal(pk_rat, gfac)
+
+
+def test_import_isitgr():
```

Comment:
You should never need a test like this or this test needs to be moved upstream to the isitgr repo. Please remove.

Reply:
I have been simply trying to figure out how these tests work for the type of error I am having. After that I will write the right needed tests (I cannot check coverall locally so have to push them here ...).  

Now, the thing I need some help with from you is how to use assert_raises  to test 
the ImportError exception in this piece of code. 

try:
    import isitgr
    import isitgr.model
except ImportError as e:
    e.args = (
        "You must have the `isitgr` python package "
        "installed to run CCL with ISiTGR-CAMB!",
        *e.args)
    raise
  
I am able to test some other types of errors fine like TypeError etc but not ImportError. Could you pitch in something?  That will same me some time.

Reply:
Try something like this:
```
    from unittest import mock
    with mock.patch.dict(sys.modules, {'isitgr':None}):
        with assert_raises(ImportError):
            get_isitgr_pk_lin(cosmo)
```

Reply:
(BTW, I completely disagree with Matt about the utility of this.  I've caught so many bugs in my life by covering lines that I thought were "obviously right".  But YMMV...)

Reply:
I will try this now.

Reply:
Thanks for the piece of code. I added it.

Reply:
The marginal cost of this bit of code going untested is tiny in comparison to say something producing wrong results. Python **will** complain if it cannot find an import. If it did, we would fix it. I'm sure we can all find better things to test besides python import statements.

Reply:
Done, anyway.